### PR TITLE
Set session vars using JDBC

### PR DESCRIPTION
**Pull request by ianjevans**
Fixes DOC-8359.

- [x] Content review
- [x] Backport to supported versions

### CODE REVIEW

File: src/current/v23.2/build-a-java-app-with-cockroachdb.md [hunk 1]
```diff
@@ -215,6 +206,48 @@ BUILD SUCCESSFUL in 8s
...
+~~~
+options=-c unbounded_parallel_scans=on
+~~~
+
+To set session variables as properties of the JDBC data source, set `options` using `setProperty`:
+
+{% include_cached copy-clipboard.html %}
+~~~ java
+Properties props = new Properties();
+props.setProperty("options", "-c {session variable name}={session variable value}");
```
Comment (fabiog1901) [inline src/current/v23.2/build-a-java-app-with-cockroachdb.md:242]: I'd add an example with 2 variables else you won't know if you need to add a comma, an additional property, or a space... this is how you do it:

```
ds.addDataSourceProperty("options", "-c unbounded_parallel_scans=true -c statement_timeout=30");
```

**Review by fabiog1901 — CHANGES_REQUESTED** [review]
added inline comment

File: src/current/v23.2/build-a-java-app-with-cockroachdb.md (hunk 1 above)
Comment (ianjevans) [reply src/current/v23.2/build-a-java-app-with-cockroachdb.md:242]: Done.

**Review by fabiog1901 — APPROVED** [review]
LGTM

**Review by dikshant — APPROVED** [review]
LGTM 🚀

**Review by rmloveland — APPROVED** [review]
LGTM!

**Pull request merged**